packages/clarity-js/types/layout.d.ts

```diff
@@ -154,6 +154,7 @@ export interface NodeValue {
     hash: [string, string];
     region: number;
     metadata: NodeMeta;
+    fragment: number;
```

Comment:
What does this represent?

Reply:
Similar to how region tracks the parent region id, fragment tracks the parent fragment id so we can do O(1) lookup for matching fragment (if any).